core/unit_test/tools/printing-tool.cpp

```diff
@@ -23,6 +32,16 @@ extern "C" void kokkosp_finalize_library() {
   std::cout << "kokkosp_finalize_library::";
 }
 
+extern "C" void kokkosp_print_help(char* exe) {
+  std::cout << "kokkosp_print_help:" << get_basename(exe) << "::";
+}
+
+extern "C" void kokkosp_parse_args(int argc, char** argv) {
+  std::cout << "kokkosp_parse_args:" << argc;
+  for (int i = 0; i < argc; ++i) std::cout << ":" << get_basename(argv[i], i);
+  std::cout << "::";
+}
```

Comment:
You should actually pass arguments to the executable.

Reply:
Yeah, so far that's my only problem. We should do this legitimately, run the test with args